internal/storage/gcs/bucket.go

```diff
@@ -32,6 +32,9 @@ const (
 	NonHierarchical
 	Hierarchical
 	Unknown
+	// ReqIdField is the key for the value of
+	// GCS req-id in context.
```

Comment:
also mention that its created by debug-bucket

Reply:
done